### DISCUSSION TITLE
[chai] expect types are inaccurate

### DISCUSSION THREAD

**Discussion by joelnwalkley** [DefinitelyTyped/DefinitelyTyped#66161] [Issues with a @types package]
The intellisense for Chai's `expect` function seem to be inaccurate. It shows most autocomplete suggestions as a properties even when they are methods. They also seem to omit the method descriptions. However, I did notice that the assert function appears correct.

Here is an example screenshot (note I have Copilot on so that is where grey suggestion is coming from):
![Screenshot 2023-07-24 at 9 09 01 AM](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/72155733/4c12cfad-c7c1-4794-bab4-3564e9811493)

Which does not match the docs for [Chai's equal](https://www.chaijs.com/api/bdd/#method_equal).

I'd be interested in contributing a solution to this issue if it is accepted.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
While `equal` is a property, it looks like it's defined as something that *is* a method:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dfa5a87c7288eac52b867bfdf0ab2b4c9b862ae1/types/chai/index.d.ts#L338-L340>

Are any errors raised when `equal` is called?

**Reply by joelnwalkley** [reply]
My tests execute correctly when using `equal` as a method, so this is focused to intellisense hints. Any idea why we don't get the method description either (even when those do seem to be defined for `assert`)?

**Comment by joeykilpatrick** [comment]
Can you confirm what version of `@types/chai` you are using?

**Reply by joelnwalkley** [reply]
`4.3.5`
